### DISCUSSION TITLE
golang/%E7%AC%AC2%E7%AB%A0%EF%BC%9A%E5%B9%B6%E5%8F%91%E7%BC%96%E7%A8%8B/16.%20%E7%BA%BF%E7%A8%8B%E4%B8%8E%E5%A4%9A%E7%BA%BF%E7%A8%8B%E7%BC%96%E7%A8%8B

### DISCUSSION THREAD

**Discussion by giscus** [xiaobaiTech/golang-full-stack#3] [Announcements]
# golang/%E7%AC%AC2%E7%AB%A0%EF%BC%9A%E5%B9%B6%E5%8F%91%E7%BC%96%E7%A8%8B/16.%20%E7%BA%BF%E7%A8%8B%E4%B8%8E%E5%A4%9A%E7%BA%BF%E7%A8%8B%E7%BC%96%E7%A8%8B

16. 线程与多线程编程 今天我们要聊聊一个有趣而又实用的主题： 线程与多线程编程 。 在我们开始之前，让我们先放轻松，别担心，这个话题并不难理解，而且对于你写出更高效的程序非常有用。 什么是线程？ 首先，让我们来解释一下什么是线程。在计算机中，线程可以看作是执行程序的一条执行路径，一个程序可以同时运行多个线程，每个线程负责不同的任务。 举个例子，想象...

http://localhost:8080/golang/%E7%AC%AC2%E7%AB%A0%EF%BC%9A%E5%B9%B6%E5%8F%91%E7%BC%96%E7%A8%8B/16.%20%E7%BA%BF%E7%A8%8B%E4%B8%8E%E5%A4%9A%E7%BA%BF%E7%A8%8B%E7%BC%96%E7%A8%8B.html

**Comment by xiaobaiTech** [comment]
test111

**Comment by xiaobaiTech** [comment]
AA
